radiomics/base.py

```diff
@@ -63,13 +61,19 @@ def __init__(self, inputImage, inputMask, **kwargs):
       self.logger.warning('Missing input image or mask')
       return
 
+  def _initLesionWiseCalculation(self):
```

Comment:
I agree with @fedorov's point about it lesion-wise sounds like per-lesion, so lesion-based would be a better fit.

But why are we using the term 'lesion' at all here?  It doesn't show up much at all in the master except in the faq and maybe we want to use a more generic term like 'segment' or 'structure'?  As we discussed with Hugo today on the call radiomics might be used to analyze organs or other structures (e.g. hippocampus).   Perhaps we can create a style guide that defines the terms 'mask' 'boundingBox' and 'segment' so we use them consistently.  Here mask and boundingBox are both aspects of the 'lesion' or 'segment' level concept (i.e. I don't think we say it explicitly but I believe we always mean: pyradiomics calculates features of segments, where segments often correspond to segmentations of lesions.)